rasa/shared/constants.py

```diff
@@ -44,7 +44,7 @@
 
 DEFAULT_NLU_FALLBACK_INTENT_NAME = "nlu_fallback"
 
-DEFAULT_E2E_TESTS_PATH = "tests"
+DEFAULT_E2E_TESTS_PATH = "."
```

Comment:
Not sure about this decision.
We don't really need this variable now since tests can be anywhere not only in `tests` directory but it might make sense to still keep this variable because it's part of `rasa.shared`.